components/AddCalories/AddCalories.js

```diff
@@ -1,7 +1,7 @@
 import useCalorieStore from "../../utils/useCalorieStore";
 import { AddCalorieButton, AddCalorieForm, FixedInput } from "./styles";
 
-export default function AddCalories() {
+export default function AddCalories({ setIsFormVisible }) {
```

Comment:
It's best practice to prefix callback props with `on` - you could e.g. call it `onHide` or `onClose` as you only call it to close the overlay and not to show it.

Reply:
I thought no changing the name would be a good Idea to increase readability